cicd/3-app/buildspec.yml

```diff
@@ -0,0 +1,29 @@
+version: 0.2
+phases:
+  install:
+    runtime-versions:
+      ruby: 2.7
+      java: corretto11
+      python: 3.8
+    commands:
+      - gem install bundler
+      - pip install cfn-lint
+  build:
+    # This should be moved to a shell script if it gets more complicated.
+    commands:
+      - set -e
+      - BRANCH_NAME=${CODEBUILD_WEBHOOK_HEAD_REF##*/}
```

Comment:
This keeps only the last item of a `/` separated string. So `refs/heads/my-branch` becomes `my-branch`. This will not work for branches with a `/` in the name, like `feature/mybranch` or `darin/mybranch`

The only reason I'm trimming it is so I can generate a path in the S3 bucket. Need a better solution than `${HEAD_REF##*/}`

Reply:
Might just be `${HEAD_REF#refs/heads/}`...